dependencies.props

```diff
@@ -45,7 +45,7 @@
   <!-- Package versions used as toolsets -->
   <PropertyGroup>
     <FeedTasksPackage>Microsoft.DotNet.Build.Tasks.Feed</FeedTasksPackage>
-    <FeedTasksPackageVersion>1.0.0-prerelease-02203-01</FeedTasksPackageVersion>
+    <FeedTasksPackageVersion>1.0.0-prerelease-02207-0</FeedTasksPackageVersion>
```

Comment:
Maestro was just merged and it will take care of this version so make sure this is merged. Same as in BuildToolsVersion.txt which you can manually update once the BuildTools official build is done and we have a new version of BuildTools